### ISSUE TITLE
Hide Advertisers from list view

### ISSUE THREAD

**Issue by robgpeak** [ChrisASearles/CompLinks#101]
@cgladue Please see if you can add the option in the "Action" menu on the page http://shop.complinks.co/admindashboard/advertisers to "Hide" Advertiser, which would take the record out of the mix on all the other four views,   JOINED   NOT JOINED   ACTIVE   INACTIVE, and put them in their own filter of HIDDEN.

It would still retain the records current state of Joined or Not Joined, and/or Active or Inactive, but it would not be visible on any filter other than Hidden.  Then of course, once the record is marked as hidden, the Action menu would have the option to "Unhide" the record.

This will be helpful in managing the advertisers that have either dropped us from their program, or we've decided at the moment not to show on our site even though we are joined, because they are offering flat commissions instead of percentages, which we are not going to address at the moment, but we may in the future.

**Comment by cgladue** [comment]
being "hidden" should also remove them from the site navigation as well i assume correct?

**Comment by robgpeak** [comment]
yes.  I think they will all be on false anyway which would prevent them from being in the site nav, but yes, hidden would do that as well.
